zero/src/intra_block_tries.rs

```diff
@@ -1,3 +1,42 @@
+//! An _Ethereum Node_ executes _transactions_ in _blocks_.
```

Comment:
To me `intra_block_tries` is a bit misleading and out of context name for a module for somebody looking at the project for the first time. `entrypoint` is not just about intermediate tries, it processes node tracer input and metadata and prepares the whole input for the prover. Not sure what would be the good alternative - e.g. `witness_processor`, `prover_input`, `preprocessor` or something else.

Reply:
For example, for somebody looking at what the leader does, `crate::prover_input::generate_inputs(block_trace, block_data)` would be much more meaningful than `crate::intra_block_tries::entrypoint()`